pkg/config/config.go

```diff
@@ -792,8 +792,12 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("logs_config.docker_client_read_timeout", 30)
 	// Internal Use Only: avoid modifying those configuration parameters, this could lead to unexpected results.
 	config.BindEnvAndSetDefault("logs_config.run_path", defaultRunPath)
+	// DEPRECATED in favor of `logs_config.force_use_http`.
 	config.BindEnvAndSetDefault("logs_config.use_http", false)
+	config.BindEnvAndSetDefault("logs_config.force_use_http", false)
```

Comment:
would it be simpler to set `force_use_http` to the same value as `use_http` when it is set in the config? I think this would allow you to remove the `or` conditions in the status and config keys and then down the road when we want to remove the deprecated setting, we can just delete the override. 

Otherwise LGTM :)

Reply:
Thanks, was thinking about something like that. I was just looking at the other deprecated fields to see if there was a pattern I could copy. Looks like `forwarder_retry_queue_max_size` might be a good example.